vulnerabilities/tests/test_api.py

```diff
@@ -647,6 +647,40 @@ def test_bulk_api_with_purl_only_option(self):
         assert len(response) == 1
         assert response[0] == "pkg:nginx/nginx@1.0.15"
 
+    def test_bulk_api_without_purls_list(self):
+        request_body = {
+            "purls": None,
+        }
+        response = self.csrf_client.post(
+            "/api/packages/bulk_search",
+            data=json.dumps(request_body),
+            content_type="application/json",
+        ).json()
+
+        expected = {
+            "error": {"purls": ["This field may not be null."]},
+            "message": "A non-empty 'purls' list of PURLs is required.",
+        }
+
+        self.assertEqual(response, expected)
+
+    def test_bulk_api_without_purls_empty_list(self):
```

Comment:
@keshav-space thanks, one test with an empty request body as well, just to verify what happens if "purls" property is not provided in the body